programs/bash-completion/completions/clickhouse-bootstrap

```diff
@@ -0,0 +1,85 @@
+#
+# bash autocomplete, that can work with:
+# a) --help of program
+#
+# Also you may like:
+# $ bind "set completion-ignore-case on"
+# $ bind "set show-all-if-ambiguous on"
+#
+# It uses bash-completion dynamic loader.
+
+# Known to work with bash 3.* with programmable completion and extended
+# pattern matching enabled (use 'shopt -s extglob progcomp' to enable
+# these if they are not already enabled).
+shopt -s extglob
+
+export _CLICKHOUSE_COMPLETION_LOADED=1
+
+function _clickhouse_bin_exist()
+{ [ -x "$1" ] || command -v "$1" >& /dev/null; }
+
+function _clickhouse_quote()
+{
+    local quoted=${1//\'/\'\\\'\'};
+    printf "'%s'" "$quoted"
+}
+function _clickhouse_get_options()
+{
+    "$@" --help 2>&1 | grep -F -a -- - \
+        | awk '{for (i=1; i<=NF; i++) print $i}' \
```

Comment:
Why the second awk by the way? I'd write something like `$BUILD/programs/clickhouse-benchmark --help | awk '{for (i=1; i<=NF; i++) print $i}' | sed -n 's/^\([-]\+[^=,<]\+\).*/\1/p'`

Reply:
Agree, it can be written better, replaced with one awk + sort